data/json/mutations.json

```diff
@@ -3897,6 +3897,16 @@
         "prereqs" : ["ROT2"],
         "category": ["MUTCAT_ALPHA"],
         "healing_awake" : -0.0008
+    },{
+        "type" : "mutation",
+        "id" : "NO_HEALING_RESTING",
+        "name" : "Lost resting healing",
```

Comment:
Suggest `[Obstructive] [Sleep Apnea/Hyponea]` as this is a major disorder that can affect homeostasis and rejuvenation.

Reply:
Being totally unable to heal any wounds at all without artificial substances is well beyond anything that could be caused by low quality of sleep.
Plus, low quality of sleep would probably cause sleep to last longer, while the mutation here doesn't really affect length of sleep.

Reply:
Unless you're going to get into some rare extreme disorder, then I believe Hypopnea offers a pretty good broad naming convention here. So following @Vasyan2006 suggestions for levels: Hypopnea -> Apnea -> Extreme Apnea. Or even more specific: Hypopnea -> Obstructive Hypopnea -> Central Hypopnea -> Apnea (person should probably losing slight HP when sleeping) -> Extreme Apnea (person should probably be losing HP when sleeping).

[And it does not increase length of sleep, in fact, it's more frequent, less restful fits of sleep](https://en.wikipedia.org/wiki/Hypopnea#Symptoms). So basically an average person sleeps for 8 hours straight, wakes up rested. Person suffering hypopnea will sleep for 3-4 hours, wake up feeling extremely tired and fatigued for period of time, try to sleep again for 3-4 hours, and repeat cycle. So they do not sleep longer, but they do sleep more frequently and not because they "need to", but when you feel tired and sleepy, you generally want to sleep/nap. So increased frequency, but not duration. [In fact, increased durations generally leads to death because they are going longer and longer without oxygen](https://en.wikipedia.org/wiki/Obstructive_sleep_apnea#Prognosis).

This is another reason why I suggested @Vasyan2006 looks at developing relationships with other sleep mutations (cancels, changes_to, etc).

Reply:
> Unless you're going to get into some rare extreme disorder, then I believe Hypopnea offers a pretty good broad naming convention here.

That's also a problem: it's too broad and implies more than we need.
For example, that 3-4 hour thing and dependency on oxygen.

> And it does not increase length of sleep, in fact, it's more frequent, less restful fits of sleep.

Mechanics of sleep:
* Character accumulates 1 point of tiredness per some time (I think 30 minutes), unless asleep
* Sleeping character "cures" 1 point of tiredness per some shorter time (I think 5 minutes)
* Character who sleeps longer gets a boost to that rest speed, up to 200% at 8 hours

End result: character who wakes up every 4 hours needs to sleep longer, ie. total length of sleep is increased.

Not to mention that it would be just plain annoying to players to require restarting sleep multiple times in the night.

Reply:
The suggested symptoms don't match any actual disorder, so I'm also against naming it after one.
If the goal is specific game effects (making healing difficult), then I'd prefer the naming to reflect that game effect.

Reply:
>The suggested symptoms don't match any actual disorder, so I'm also against naming it after one.
If the goal is specific game effects (making healing difficult), then I'd prefer the naming to reflect that game effect.

[Those symptoms are for more generalized sleep apnea, as the terms are basically interchangeable](https://en.wikipedia.org/wiki/Hypopnea)
>Hypopnea is defined by some to be less severe than apnea (the complete cessation of breathing), while other researchers have discovered hypopnea to have a "similar if not indistinguishable impact" on the negative outcomes of sleep breathing disorders.

[Symptoms for Hypopnea/Sleep Apnea](https://www.mayoclinic.org/diseases-conditions/sleep-apnea/symptoms-causes/syc-20377631):
>    Loud snoring, which is usually more prominent in obstructive sleep apnea
    Episodes of breathing cessation during sleep witnessed by another person
    Abrupt awakenings accompanied by shortness of breath, which more likely indicates central sleep apnea
    Awakening with a dry mouth or sore throat
    Morning headache
    Difficulty staying asleep (insomnia)
    Excessive daytime sleepiness (hypersomnia)
    Attention problems
    Irritability

[Other complications](https://en.wikipedia.org/wiki/Sleep_apnea#Signs_and_symptoms) (OSA = Obstructive Sleep Apnea):
> excessive daytime sleepiness (EDS), impaired alertness, and vision problems. OSA may increase risk for driving accidents and work-related accidents. If OSA is not treated, people are at increased risk of other health problems, such as diabetes. Death could occur from untreated OSA due to lack of oxygen to the body.
Due to the disruption in daytime cognitive state, behavioral effects may be present. These can include moodiness, belligerence, as well as a decrease in attentiveness and energy. These effects may become intractable, leading to depression.
There is evidence that the risk of diabetes among those with moderate or severe sleep apnea is higher. There is increasing evidence that sleep apnea may lead to liver function impairment, particularly fatty liver diseases (see steatosis). Finally, because there are many factors that could lead to some of the effects previously listed, some people are not aware that they have sleep apnea and are either misdiagnosed or ignore the symptoms altogether.

And in terms of [harming a sleeping survivor](https://en.wikipedia.org/wiki/Apnea#Apneic_oxygenation) [emphasis mine]:
>However, no CO2 is removed during apnea. The partial pressure of CO2 in the airspace of the lungs will quickly equilibrate with that of the blood. As the blood is loaded with CO2 from the metabolism, more and more CO2 will accumulate and eventually displace oxygen and other gases from the airspace. CO2 will also accumulate in the tissues of the body, resulting in **respiratory acidosis**.